site/content/docs/user/quick-start.md

```diff
@@ -242,6 +242,49 @@ nodes:
 - role: worker
 ```
 
+### Enable Feature Gates in Your Cluster
+
+Feature gates are a set of key=value pairs that describe alpha or experimental features. In order to enable a gate you have to [customize your kubeadm configuration][customize control plane with kubeadm], and it will depend on what gate and component you want to enable. An example kind config can be:
+
+```
+# this config file contains all config fields with comments
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+# patch the generated kubeadm config with some extra settings
+kubeadmConfigPatches:
+- |
+  apiVersion: kubeadm.k8s.io/v1beta1
+  kind: ClusterConfiguration
+  metadata:
+    name: config
+  apiServer:
+    extraArgs:
+      "feture-gates": "FeatureGateName=true"
```

Comment:
Is it really spelled `feture-gates`? I haven't found anywhere in k/k this way of writing **feature**...

Reply:
great catch, thanks